showcase/image-analysis.md

```diff
@@ -0,0 +1,56 @@
+affiliations: [European Union, Euro-BioImaging, ISIDORe]
+page_id: image_analysis_isidore
+
+---
+
+## Introduction 
+
+Coronavirus disease 2019 (Covid-19) has been linked to a range of neurological conditions, including neuroinflammation and impaired energy metabolism in the brain. In the field of neuroscience, a variety of tracers are frequently employed to investigate diverse aspects of brain health and dysfunction. Indeed, recent articles have demonstrated the presence of pronounced inflammatory processes in the brains of long COVID patients, by the use of inflammatory and metabolic tracers in positron emission tomography (PET) imaging (1, 2) and Autoradiography (ARG).
+This showcase details the development, of an open-source, user-friendly, automated analysis pipeline for aligning and processing autoradiography images from mouse brain. Funded by the [ISIDORe project](https://isidore-project.eu/), his so called {% tool "mbat" %} facilitates enhanced efficiency, accuracy, and reproducibility in the registration and analysis of images in studies aiming to elucidate the characteristics and underlying mechanisms of neuroinflammation due to infection or other causes.
+
+
+## Who is the showcase intended for?
+
+The MBAT tool is important for life science researchers working with autoradiographic data of the mouse brain. Thematically, this analysis has been specifically designed to study neuronal changes and neuroinflammation due to (viral) infection, but is also broadly applicable to all other neuroscience topics.
+Since the tool is built to be user-friendly, researchers with minimal knowledge of scientific programming can use and greatly benefit from the tool. Being open source, the pipeline can be easily adapted to individual research projects by researchers with more in-depth, advanced bioinformatics skills. The analyzed brain regions from the pipeline can be also of interest to AI and computer scientists.
+
+
+## What is the showcase?
+
+{% include image.html file="/MBAT_scheme.png" caption="Figure 1. Development of an open source, semi-automated processing pipeline for aligning autoradiography images from mouse brain tissue to study infection-induced neuroinflammation. The python-based pipeline takes input from data generated through Euro-BioImaging and is itself developed by funding from the ISIDORe project and allows users to align the images in a Napari-based user interface to the Allen Brain Atlas. The full code is open source available on GitHub." alt="Image with a central symbolic computer screen with the program MBAT. Arrows flowing in and out of the screen denoting inputs (data generated trough Euro-Bioimaging, Funding from ISIDORe project, Tool development wiht Python and Napari) and outputs (Analyzed data and Code deposition on GitHub) of the software." %}
```

Comment:
- references needed for: Napari-based user interface to the Allen Brain Atlas.
- typo: data generated *trough* Euro-Bioimaging

Reply:
done